components/prize_pool/wikis/overwatch/prize_pool_custom.lua

```diff
@@ -40,6 +41,18 @@ function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
 		Variables.varDefault('tournament_type')
 	)
 
+	local participantLower = mw.ustring.lower(lpdbData.participant)
+
+	Variables.varDefine(participantLower .. '_prizepoints', lpdbData.extradata.prizepoints)
+	Variables.varDefine('enddate_'.. lpdbData.participant .. '_date', lpdbData.date)
+	Variables.varDefine('status'.. lpdbData.participant .. '_date', lpdbData.date)
+
+	lpdbData.qualified = placement:getPrizeRewardForOpponent(opponent, 'QUALIFIES1') and 1 or 0
```

Comment:
Prefer not setting this one unless absolutely needed, is it needed or just a copy-paste?

Reply:
Just a copy-paste

Reply:
Then I would ask to only keep what is necessary, instead of everything